staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -371,6 +389,13 @@ func (req *request) Finish(execFn func()) bool {
 	return idle
 }
 
+func (req *request) OnDispatch() {
+	qs := req.qs
+	if qs != nil {
+		qs.totRequestsDispatched++
```

Comment:
This field should only be accessed while the qs mutex is held.

Reply:
Fixed.